### PR TITLE
Qhc 24 write 1 demo about how to use qililab

### PR DESCRIPTION
**Pull request by visagim**

### CODE REVIEW

**Comment by visagim** [comment]
Creating pr so we can discuss changes / additions / edits

**Comment by visagim** [comment]
Still missing:
- Running the notebook on the chip so we can get the rabi plot and add markdown cell below the plot explaining which amplitude value we select for the pi-pulse

File: demos/rabi.ipynb [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+   "source": [
+    "### Rabi Experiment"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A Rabi experiment aims to find the $\\pi$ pulse amplitude that will help you exciting your qubit from \n",
```
Comment (AlbertMitjans) [inline demos/rabi.ipynb:27]:
```suggestion
    "A Rabi experiment aims to find the $\\pi$ pulse amplitude that will help you excite your qubit from \n",
```

File: demos/rabi.ipynb [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "from qibo import gates\n",
+    "from qibo.models import Circuit\n",
+    "from tqdm.auto import tqdm\n",
+    "\n",
+    "import qililab as ql\n",
+    "\n",
+    "ql.logger.setLevel(40)"
```
Comment (AlbertMitjans) [inline demos/rabi.ipynb:109]: Explain why we add this line (you can add a comment inside the code)

File: demos/rabi.ipynb [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+   "source": [
+    "Next we are going to define some constants our experiment will be using. In order to calibrate the amplitude for our $\\pi$ pulse, \n",
+    "we need to loop over a subset of amplitudes around which one we expect to find the right amplitude to drive our qubit from the \n",
+    "ground state, $| 0 \\rangle$, to the excited state, $| 1 \\rangle$.\n",
+    "\n",
+    "First, we need to indicate Qililab which qubit, from our chip, we want to interact with. We will need to define the amplitude \n",
+    "values array, for which we will use the numpy function linspace, wrapping the initial and final amplitude values, and the number of \n",
+    "steps to generate in between. \n",
+    "\n",
+    "The hardware average constant allow us to specify the average of values for the results of the experiment. The repetition duration\n",
```
Comment (AlbertMitjans) [inline demos/rabi.ipynb:133]: Unfinished sentence

File: demos/rabi.ipynb [hunk 4]
```diff
@@ -0,0 +1,299 @@
...
+    "\n",
+    "Finally, we need to specify where our runcard is located so Qililab can instatiate our laboratory setup correctly. The runcard contains all necessary information about our laboratory setup and the chip calibrations which we need to run quantum circuits."
```
Comment (AlbertMitjans) [inline demos/rabi.ipynb:135]: I don't understand the end of this sentence. What are "the chip calibrations"?

**Review by AlbertMitjans — COMMENTED** [review]
This demo looks awesome Victor! Good job!! 🚀

File: demos/images/bloch.gif
Comment (GuillermoAbadLopez) [inline demos/images/bloch.gif:1]: Looking 🔥🔥🔥!

**Review by AlbertMitjans — COMMENTED** [review]
Looks good! Waiting for the data 😄

**Review by AlbertMitjans — APPROVED** [review]
Looks amazing! Thanks Victor!

**Pull request merged**